src/index.html

```diff
@@ -11,5 +11,51 @@
     <header class="app-header">
       <h1 class="app-header__heading">Quiz App</h1>
     </header>
+    <main>
+      <section class="quiz-card">
+        <h3 class="quiz-card__question">Question</h3>
+        <button class="quiz-card__button">Show Answer</button>
+        <p class="quiz-card__answer">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Animi, quas.
+        </p>
+        <ul class="quiz-card__tag-list">
+          <li>tag</li>
+          <li>tag</li>
+          <li>tag</li>
+          <li>tag</li>
+        </ul>
+        <div class="quiz-card__bookmark"></div>
```

Comment:
You could give this the attribute `role="button"` 
```suggestion
        <div role="button" class="quiz-card__bookmark"></div>
```